oci/utils_unix_test.go

```diff
@@ -149,9 +150,11 @@ func TestHostDevicesAllValid(t *testing.T) {
 	}
 
 	for _, device := range devices {
-		// Devices can't have major number 0.
-		if device.Major == 0 {
-			t.Errorf("device entry %+v has zero major number", device)
+		if runtime.GOOS != "freebsd" {
+			// On Linux, devices can't have major number 0.
```

Comment:
Nit: should it be GOOS == linux rather than != freebsd?

Reply:
I'm not sure.  The file right now is built for `!windows && !darwin` and I know this assertion won't apply to FreeBSD, but I'm not sure about other Unix implementations (and whether anyone is actively looking at a Solaris/Illumos, OpenBSD, or NetBSD port of containerd).  I'm happy to change this to GOOS == linux if you think that's more appropriate here.

Reply:
I see. The condition and the comment below are not aligned, but I'm fine to keep that as is.